web/src/encoding.ts

```diff
@@ -57,19 +57,19 @@ const KeggPrefix: Record<string, PrefixInfo> = {
     pattern: /^((k:?)|(kegg\.orthology:k))(?=\d{5})/i,
     short: () => 'k',
     long: () => 'KEGG.ORTHOLOGY:K',
-    urlBase: 'https://www.genome.jp/entry/',
+    urlBase: 'https://bioregistry.io/kegg.orthology:/',
```

Comment:
I think we might need to force uppercase for the K
![image](https://github.com/user-attachments/assets/ff9aaed2-a3b1-4a2e-a910-68f09fa45272)